src/encoded/types/utils.py

```diff
@@ -0,0 +1,26 @@
+
```

Comment:
are there other functions in types/* that could be moved here?

Reply:
Hmm. I think I might just put these in https://github.com/ENCODE-DCC/snovault/blob/dev/src/snovault/util.py